src/pip/_internal/req/req_install.py

```diff
@@ -559,10 +559,11 @@ def egg_info_path(self, filename):
                 filenames = [f for f in filenames if f.endswith('.egg-info')]
 
             if not filenames:
-                error_message="Files/directories (from %s) " + \
-                    "not found in %s " % (filename, base)
-                raise InstallationError(error_message)
-            assert filenames, error_message
+                raise InstallationError(
+                    "Files/directories (from %s) " +
+                    "not found in %s" % (filename, base))
+            assert filenames, "Files/directories (from %s) " + \
```

Comment:
remove the assert?

Reply:
Done and pushed. Do I need to resubmit the PR or will it just read the update?